plugins/sf2_player/patches_dialog.h

```diff
@@ -65,10 +65,10 @@ protected slots:
 
 protected:
 
-	void setBankProg(int iBank, int iProg);
+	void setBankProg( int iBank, int iProg );
 
-	QTreeWidgetItem *findBankItem(int iBank);
-	QTreeWidgetItem *findProgItem(int iProg);
+	QTreeWidgetItem *findBankItem( int iBank );
+	QTreeWidgetItem *findProgItem( int iProg );
```

Comment:
The formatting here should be left as-is - we're moving over to this style, without the spaces between the parentheses.

Reply:
I will amend the commits about these.

Btw do you have a .clang-format file I can use to format automatically? I hate to turn off format on save just for LMMS and turn it back on for every other project.

Reply:
There's #4690, but I have no idea whether it's up to date with our current [coding conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions). It might be a nice opportunity to test it though.

Reply:
The file in #4690 seems to tick all the boxes, however I cannot use it right now because the old files are not conform to such standards in the first place. Good to see it moving along though.